nidirect_gp/src/Controller/GpSearchController.php

```diff
@@ -91,14 +99,17 @@ class GpSearchController extends ControllerBase {
    *   Geocoding service ID.
    * @param \Drupal\core\Form\FormBuilder $form_builder
    *   Form builder.
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager
+   *   Entity type manager.
    */
   public function __construct(
     RequestStack $request_stack,
     PostcodeExtractor $postcode_extractor,
     GeocoderInterface $geocoder,
     int $proximity_max_distance,
     string $geocoding_service_id,
-    FormBuilder $form_builder
+    FormBuilder $form_builder,
```

Comment:
FormBuilderInterface would be ideal here as it means you can pass in a test class that implements that, if needed.

Reply:
OK, have done that now